nginx-app/templates/default/silex.conf.erb

```diff
@@ -7,7 +7,7 @@ server {
 
     index <%=@default_router%> index.php;
 
-    access_log <%=@node.fetch('nginx-app', {}).fetch('access_log', {}) %>;
+    access_log <%= node[:nginx-app][:access_log] %>;
```

Comment:
I think this will not work, you cannot have a "symbol" with a "-" in it. I'll take a look at this in a little bit.